tests/test_utils/test_testcase.py

```diff
@@ -1,11 +1,17 @@
 from django.db import IntegrityError, connections, transaction
 from django.test import TestCase, skipUnlessDBFeature
+from django.utils.deprecation import RemovedInDjango40Warning
 
 from .models import Car, PossessedCar
 
 
 class TestTestCase(TestCase):
```

Comment:
Can we also have a test/testcase of a deepcopy'd pair of objects that refer to each other? Doesn't seem to be explicitly tested anywhere.

Reply:
I feel like this would be testing `copy.deepcopy`/`pickle` behaviour which might not be necessary here? Happy to add a test if mergers deem it necessary.

Reply:
Imo it would test that you correctly set and get the memo dictionary

Reply:
Isn't [`TestDataTests.test_known_related_objects_identity_preservation`](https://github.com/django/django/pull/12608/files#diff-986c55ac2777dd64d0fc5e45a6b1b2cdR100-R103) testing that?

Reply:
👀😳

Reply:
Resolving since I think we agree it's tested now 😅